src/libs/actions/IOU.ts

```diff
@@ -293,7 +292,10 @@ function getReportPreviewAction(chatReportID: string, iouReportID: string): Onyx
     // Find the report preview action from the chat report
     return (
         Object.values(reportActions).find(
-            (reportAction) => reportAction && reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.REPORT_PREVIEW && reportAction.originalMessage.linkedReportID === iouReportID,
+            (reportAction) =>
+                reportAction &&
+                reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.REPORT_PREVIEW &&
+                (reportAction as ReportAction & OriginalMessageReportPreview).originalMessage.linkedReportID === iouReportID,
```

Comment:
Why do we need assertion here? I think the type is narrowed down by the check of `actionName` above
```suggestion
                reportAction.originalMessage.linkedReportID === iouReportID,
```